### PR TITLE
Fix ui resize issue to grow accordingly

### LINKED ISSUE #122: Fix UI layout when stretching

### PR DESCRIPTION
**Pull request by NorbertLoh**
Fixes #122.

### CODE REVIEW

File: src/main/resources/view/MainWindow.fxml [hunk 1]
```diff
@@ -32,8 +32,8 @@
                         <MenuItem fx:id="helpMenuItem" mnemonicParsing="false" onAction="#handleHelp" text="Help"/>
                     </Menu>
                 </MenuBar>
-                <HBox>
-                    <VBox>
+                <HBox VBox.vgrow="ALWAYS">
+                    <VBox VBox.vgrow="ALWAYS" HBox.hgrow="ALWAYS">
```
Comment (Ma-Xizi) [inline src/main/resources/view/MainWindow.fxml:36]: HBox.hgrow ensures that the app can expand horizontally

File: src/main/resources/view/MainWindow.fxml [hunk 2]
```diff
@@ -55,7 +55,8 @@
                             <StackPane fx:id="personListPanelPlaceholder" VBox.vgrow="ALWAYS"/>
                         </VBox>
                     </VBox>
-                    <VBox minWidth="250" prefWidth="250" styleClass="pane-with-border">
+                    <VBox minWidth="250" maxWidth="250" prefWidth="250" styleClass="pane-with-border"
+                          VBox.vgrow="ALWAYS">
```
Comment (Ma-Xizi) [inline src/main/resources/view/MainWindow.fxml:59]: VBox.vgrow ensures that the iterface should grow vertically to fill any available extra space

**Comment by Ma-Xizi** [comment]
LGTM!

**Review by Ma-Xizi — APPROVED** [review]

**Review by ethantyh692 — APPROVED** [review]
LGTM!

**Pull request merged**